packages/faustwp-cli/src/healthCheck/validateFaustEnvVars.ts

```diff
@@ -1,5 +1,15 @@
 import { getWpSecret } from '../utils/index.js';
-import { errorLog, warnLog } from '../stdout/index.js';
+import { errorLog, infoLog, warnLog } from '../stdout/index.js';
+
+export function isWPEngineComSubdomain(url: string) {
+  const regex = /\b\w+\.wpengine\.com\b/;
+
+  if (regex.test(url)) {
```

Comment:
```suggestion
 return regex.test(url);
```

Reply:
Since regex.test(url) is true if it matches you can just return the value here.

Reply:
@theodesp Good call